src/components/Collisions.ts

```diff
@@ -1,421 +1,427 @@
-import { GeneralComponent } from "gamestartr";
```

Comment:
Try resetting these files to master by copy & pasting the raw content from GitHub, then reapplying your changes on top of them.

Reply:
Also:
https://github.com/atom/git-diff/issues/127
https://github.com/Microsoft/WSL/issues/184